hypothesis-python/src/hypothesis/strategies/_internal/types.py

```diff
@@ -59,6 +59,21 @@
 except ImportError:
     _GenericAlias = ()
 
+try:
+    from typing_extensions import _AnnotatedAlias  # type: ignore
```

Comment:
```suggestion
    from typing_extensions import _AnnotatedAlias
```

Checking the CI logs, apparently we can remove some of these ignores!